medical-clinic/src/main/java/com/Mafiuz04/medicalclinic/PatientService.java

```diff
@@ -0,0 +1,64 @@
+package com.Mafiuz04.medicalclinic;
+
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class PatientService {
+    List<Patient> patients = new ArrayList<>();
+
+    public void showAllPatients() {
```

Comment:
ta metoda powinna sluzyc do zwrócenia pacjentów a nie ich wyspisania. 
W takim razie powinna zwracac List<Patient>.

Czy powinnismy zrobic ze ona zwraca :

return patients? <- nie
dlaczego nie? Ponieważ jeśli zwrócilibyśmy całą listę to osoba, która taką liste dostała może nią manipulować (usuwac z niej pacjentow, dodawac do niej pacjentow bez uzycia naszych metod, ktore do tego sluzą). 

Powinniśmy zwrócić kopię tej listy.

Reply:
nazwa metody: getPatients